### DISCUSSION TITLE
The connection connects but than after a minute disconnects

### DISCUSSION THREAD

**Discussion by hauserpro** [IDmedia/hass-unraid#7] [Q&A]
Hi, thank you for the brilliant app. I just installed has-unraid. I installed the docker and configured everything. It seems to work and connects to HA. I see the unraid server under MQTT. But for whatever reason it disconnects only after about 1 minute or so and stays disconnected until i restart the hass-agent VM. Can you please give me some pointers on where i can logo for the problem?

**Comment by Kuchar09** [comment]
Same issue! Any luck resolving it?

I see these messages in the MQTT server

1724299422: Client Server has exceeded timeout, disconnecting.
1724299422: Socket error on client Server, disconnecting.
1724299422: Sending PUBLISH to 4lyOP5Vy8nXRjoPceLozbM (d0, q0, r0, m0, 'unraid/server/connectivity/state', ... (3 bytes))

**Comment by IDmedia** [comment]
https://github.com/IDmedia/hass-unraid/pull/13
I hope this resolves the issue. Please let me know if it persists
